wp-content/plugins/wp20-meetup-events/wp20-meetup-events.js

```diff
@@ -428,12 +429,33 @@ var WP20MeetupEvents = app = ( function( $ ) {
 		return new MarkerClusterer( app.map, markersArray, clusterOptions );
 	}
 
+	/**
+	 * Handle user input in the filter form.
+	 *
+	 * @param {object} event
+	 */
+	function handleFilterInput( event ) {
+		event.preventDefault();
+
+		filterEventList( this.value );
+
+		_.debounce( function() {
```

Comment:
My guess is this will likely not work and will be called numerous times. Try adding a `console.log` here to debug.

You'll need to debounce in the `keyup` function.

Reply:
Yeah I had the same thought. Debouncing the whole thing stopped the list filtering from happening though. I didn't dig any deeper, but might need 2 listeners, one debounced, one not.

Reply:
Ah, ok. The scrolling looked smooth on my machine, but `console.log` shows the problem, thanks! I've made a few changes that I think will fix it. How does it look to you now?

Reply:
Hmm I still get the jumping in Chrome, but not Firefox. I think it's minor enough to ignore tbh, and we can iterate if need be.

Worth noting that on a real device the keyboard is shown so I actually can't see the results, which might be a bigger issue.

<img src="https://user-images.githubusercontent.com/1017872/231309678-51c3ad3f-de16-4894-a7e4-e7942d29a011.jpg" width="300" />

Another post launch issue to discuss with design?

Reply:
Ah, I hadn't thought of mobile keyboards, that's a good point. That dovetails into #54, but I agree that these are smaller issues and it'd be good to iterate post-launch rather than blocking.

I'm not seeing the jumping in Chrome either, but I'll open an issue where we can figure that out.

Reply:
I opened #61 and #62 for those.